_includes/site-header.html

```diff
@@ -5,7 +5,7 @@
     <div class="usa-logo" id="extended-mega-logo">
       <em class="usa-logo__text">
         <a href="/" title="Home" aria-label="Home">
-          <img src="assets/img/DHS_4R_Blue.svg" alt="Department of Homeland Security logo" width="247" height="72">
+          <div class="usa-sr-only">Department of Homeland Security</div>
```

Comment:
4. Changed the logo from an inline image because we'll have more control as a background image on different breakpoints.